README.md

```diff
@@ -12,6 +12,18 @@ rake db:create db:migrate
 # running
 
 have redis and postgres running
-in one tab: `./bin/cable`
-in another tab: `./bin/rails s`
 
+```bash
+./bin/rails s
+# OR
+bundle exec puma -p 3000 config.ru
+# OR
+foreman run web
+```
```

Comment:
Why don't we keep a single way to run the app, so anyone looking at the example wouldn't be confused? I'd prefer using `bin/rails s` because even beginners who aren't familiar with puma or foreman will not be confused. What do you think?